tests/colr_to_svg.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nanoemoji import colors
+from nanoemoji import color_glyph
+from picosvg import svg_meta
+from picosvg.svg import SVG
+from fontTools import ttLib
+from picosvg.geometric_types import Rect
+import test_helper
+from lxml import etree
+from typing import Tuple
+from fontTools.pens import svgPathPen
+from fontTools.pens import transformPen
+from fontTools.ttLib.tables import otTables
+
+
+_PAINT_SOLID = 1
+_PAINT_LINEAR_GRADIENT = 2
+_PAINT_RADIAL_GRADIENT = 3
+
+
+def _emsquare_to_viewbox(upem: int, view_box: Rect):
+    if view_box != Rect(0, 0, view_box.w, view_box.w):
+        raise ValueError("We simply must have a BOX from 0,0")
+    return color_glyph.map_viewbox_to_emsquare(Rect(0, 0, upem, upem), view_box.w)
```

Comment:
the viewbox might not be square.

how about :

```
Affine2D.rect_to_rect(Rect(0, 0, upem, upem), Rect(view_box.x, view_box.y + view_box.h, view_box.w, -view_box.h)
```

(untested)

Reply:
this is even better `Affine2D.rect_to_rect(Rect(0, upem, upem, -upem), view_box)`